DQM/EcalCommon/src/MESetEcal.cc

```diff
@@ -214,6 +214,8 @@ namespace ecaldqm {
       if (!me)
         throw_("ME could not be booked");
 
+      assert(me);  // Silence llvm-analyzer warnings, actual check is done above.
```

Comment:
I think this type of warning you can fix by adding ` __attribute__((analyzer_noreturn))` to the function which does not return. So in this case add ` __attribute__((analyzer_noreturn))` to https://github.com/cms-sw/cmssw/blob/master/DQM/EcalCommon/interface/MESet.h#L149

Reply:
I see we have used `[[noreturn]]` alot (https://github.com/search?q=repo%3Acms-sw%2Fcmssw%20noreturn&type=code ), so add `[[noreturn]]` to `throw_` should fix the clang analyzer warning